### PR TITLE
sbcl 1.2.9

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by bfontaine** [comment]
Mmmh it fails with a warning:

```
WARNING! Some of the contrib modules did not build successfully or pass
their self-tests. Failed contribs:"
  asdf
```

File: Library/Formula/sbcl.rb [hunk 1]
```diff
@@ -96,15 +94,14 @@ def install
     end
 
     ENV["INSTALL_ROOT"] = prefix
-    system "sh install.sh"
+    sh "install.sh"
```
Comment (jacknagel) [inline Library/Formula/sbcl.rb]: Using `sh` (from FileUtils) has different semantics that using `system` (which is specialized for formulae). So this should be `system "sh", "install.sh"`.

Comment (bfontaine) [inline Library/Formula/sbcl.rb]: @jacknagel Thanks, fixed.

**Comment by mikemcquaid** [comment]
Installation failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20790/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_sbcl/

**Comment by asparagui** [comment]
This (sbcl-internal) asdf test has been failing since September.  I tried a bunch of ways of fixing it, to no avail.  Good luck!

**Comment by bfontaine** [comment]
The weird thing it that (despite some warnings) it works on my Mac:

```
(…)
The build seems to have finished successfully, including       18 (out of       18)
contributed modules.
(…)
SBCL has been installed:
 binary /usr/local/Cellar/sbcl/1.2.8/bin/sbcl
 core and contribs in /usr/local/Cellar/sbcl/1.2.8/lib/sbcl/

Documentation:
 man /usr/local/Cellar/sbcl/1.2.8/share/man/man1/sbcl.1
cp: doc/manual/*.info: No such file or directory
cp: doc/manual/*.info-*: No such file or directory
cp: doc/manual/*.pdf: No such file or directory
cp: doc/manual/sbcl.html: No such file or directory
cp: doc/manual/asdf.html: No such file or directory
==> Cleaning
rmdir: /usr/local/Cellar/sbcl/1.2.8/share/info (empty)
rmdir: /usr/local/Cellar/sbcl/1.2.8/share/doc/sbcl/html (empty)
rmdir: /usr/local/Cellar/sbcl/1.2.8/lib/sbcl/site-systems (empty)
==> Finishing up
ln -s ../Cellar/sbcl/1.2.8/bin/sbcl sbcl
ln -s ../../Cellar/sbcl/1.2.8/share/doc/sbcl sbcl
ln -s ../../../Cellar/sbcl/1.2.8/share/man/man1/sbcl.1 sbcl.1
ln -s ../Cellar/sbcl/1.2.8/lib/sbcl sbcl
==> Summary
🍺  /usr/local/Cellar/sbcl/1.2.8: 48 files, 56M, built in 6.8 minutes
```

**Pull request closed without merging**